cnap/core/pipeline.py

```diff
@@ -116,27 +114,53 @@ def unregister_pipeline(self, pipeline_id: str) -> None:
         LOG.debug("Unregister Pipeline: %s", pipeline_id)
         self._db.del_table_object(PipelineManager.PIPELINE_TABLE, pipeline_id)
 
-    def set_infer_fps(self, pipeline_id: str, infer_info_id: str, infer_fps: int) -> None:
+    def set_infer_fps(self, pipeline_id: str, infer_info: InferenceInfo, infer_fps: int) -> None:
         """Set inference fps for a pipeline.
 
         Args:
             pipeline_id (str): The id of pipeline to set inference fps.
+            infer_info (InferenceInfo): The inference info to set inference fps.
             infer_fps (int): The inference fps to set.
 
         Raises:
             ValueError: Propagates the ValueError raised by `get_table_object_dict`
                 or `save_table_object_dict` if some cases are met.
         """
+        infer_info_id = infer_info.id
```

Comment:
why not directly use it in the following code,

Reply:
Fixed, thanks